temporalio/client.py

```diff
@@ -287,6 +287,7 @@ async def start_workflow(
         start_signal_args: Sequence[Any] = [],
         rpc_metadata: Mapping[str, str] = {},
         rpc_timeout: Optional[timedelta] = None,
+        eager_start: bool = False,
```

Comment:
```suggestion
        request_eager_start: bool = False,
```

I wonder if this is a bit clearer (or "enable eager start" or "suggest eager start" or something that doesn't imply any guarantees)

Reply:
Good point about not implying guarantees, changing it...